src/Aspire.Hosting.Azure.AIFoundry/AIFoundryModel.Local.Generated.cs

```diff
@@ -10,15 +10,20 @@ public partial class AIFoundryModel
     /// <summary>
     /// Models available on Foundry Local.
     /// </summary>
-    public static class Local
+    public static partial class Local
     {
         /// <summary>
-        /// This model is an optimized version of DeepSeek-R1-Distill-Qwen-14B to enable local inference. This model uses RTN quantization. # Model Description - **Developed by:** Microsoft - **Model type:** ONNX - **License:** MIT - **Model Description:** This is a conversion of the DeepSeek-R1-Distill-Qwen-14B for local inference. - **Disclaimer:** Model is only an optimization of the base model, any risk associated with the model is the responsibility of the user of the model. Please verify and test for your scenarios. There may be a slight difference in output from the base model with the optimizations applied. Note that optimizations applied are distinct from fine tuning and thus do not alter the intended uses or capabilities of the model. # Base Model Information See Hugging Face model [DeepSeek-R1-Distill-Qwen-14B](https://huggingface.co/deepseek-ai/DeepSeek-R1-Distill-Qwen-14B) for details.
+        /// This model is an optimized version of DeepSeek-R1-Distill-Qwen-1.5B to enable local inference on Intel GPUs. # Model Description - **Developed by:** Microsoft - **Model type:** ONNX - **License:** MIT - **Model Description:** This is a conversion of the DeepSeek-R1-Distill-Qwen-1.5B for local inference on Intel GPUs. - **Disclaimer:** Model is only an optimization of the base model, any risk associated with the model is the responsibility of the user of the model. Please verify and test for your scenarios. There may be a slight difference in output from the base model with the optimizations applied. Note that optimizations applied are distinct from fine tuning and thus do not alter the intended uses or capabilities of the model. # Base Model Information See Hugging Face model [DeepSeek-R1-Distill-Qwen-1.5B](https://huggingface.co/deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B) for details.
         /// </summary>
-        public static readonly AIFoundryModel DeepseekR114b = new() { Name = "deepseek-r1-14b", Version = "3", Format = "Microsoft" };
+        public static readonly AIFoundryModel DeepseekR115b = new() { Name = "deepseek-r1-1.5b", Version = "1", Format = "Microsoft" };
```

Comment:
nit: I think it would be better if code gen added new models at the end of the file instead of the start. That would minimize PR diffs

Reply:
@JamesNK thanks. That would mean keep some state in the repos, or parse the existing file because we'd have to detect they are new. But doable. And then we could have them reordered in a different PR as a different tool.

Reply:
Maybe sort alphabetically?